src/colors/blue.js

```diff
@@ -25,5 +25,7 @@ export default {
   blue_70: '#1f5dc2',
   blue_80: '#164ea8',
   blue_90: '#114091',
-  blue_100: '#0f397d'
+  blue_100: '#0f397d',
+
+  a11y_text_light: [60, 70, 80, 90, 100]
```

Comment:
This should still be a Set. IE11 has the add method just not the Array constructor. We can polyfill or use reduce

[1, 2, 3].reduce(function(acc, v) {acc.add(v); return acc;}, new Set())

Reply:
We may optimize this at a later date, but shipping as-is for now.